src/explore-education-statistics-frontend/src/modules/data-catalogue/__tests__/DataSetFilePage.test.tsx

```diff
@@ -48,6 +48,26 @@ describe('DataSetFilePage', () => {
         name: /National statistics/,
       }),
     ).toBeInTheDocument();
+    expect(
+      within(screen.getByTestId('Geographic levels-value')).getByText(
```

Comment:
I feel like there's a preferred way of getting the values from a `dt` - you might wanna ask Nick!

Reply:
Nick said remove the `-value` so it's gone

Reply:
Cooool